docs/rules/sort-amd-paths.md

```diff
@@ -16,24 +16,31 @@ Determines which part (or parts) of the path will be used to sort by. The possib
 * `fullpath` - Sorts by the full path, including the module file name.
 * `basename` - Sorts by module file name only, ignoring the directory name.
 
+#### `sortPlugins`
+
+Determines how paths with plugin prefix will be sorted. The possible values are:
+
+* `preserve` (default) - Behaves like there is no plugin prefix before the path. Prefix is a part of a path.
+* `ignore` - Behaves like there is no plugin prefix before the path. Plugin prefix is not a part of a path.
+* `first` - Paths with plugin prefix will be sorted before non-prefixed paths.
+* `last` - Paths with plugin prefix will be sorted after non-prfixed paths.
+
 #### `ignoreCase`
 
 Accepts a boolean value. Determines whether the comparison should be case-sensitive or not. The default is `true`.
 
 ```json
-    "requirejs/sort-amd-paths": 2
-    "requirejs/sort-amd-paths": [2, { "compare": "dirname-basename" }]
-    "requirejs/sort-amd-paths": [2, { "compare": "dirname-basename", "ignoreCase": true }]
-    "requirejs/sort-amd-paths": [2, { "compare": "dirname-basename", "ignoreCase": false }]
-    "requirejs/sort-amd-paths": [2, { "compare": "fullpath" }]
-    "requirejs/sort-amd-paths": [2, { "compare": "fullpath", "ignoreCase": true }]
-    "requirejs/sort-amd-paths": [2, { "compare": "fullpath", "ignoreCase": false }]
-    "requirejs/sort-amd-paths": [2, { "compare": "basename" }]
-    "requirejs/sort-amd-paths": [2, { "compare": "basename", "ignoreCase": true }]
-    "requirejs/sort-amd-paths": [2, { "compare": "basename", "ignoreCase": false }]
+"requirejs/sort-amd-paths": [
+    2,
+    {
+        "compare": "dirname-basename | fullpath | basename",
```

Comment:
Since this is not valid JSON syntax, can we only include one option for each attr in the examples? Ideally they should be directly copy-pastable without modification